proposals/graduation/spiffe_spire.md

```diff
@@ -0,0 +1,128 @@
+
+# SPIFFE/SPIRE Graduation Proposal
+
+  
+
+The SPIFFE and SPIRE projects were started with an ambitious vision: to automatically provide every workload with an identity that is cryptographically verifiable, automatically rotated, interoperable and portable across a wide variety of platforms and providers. The goal is an interoperable workload identity format (SPIFFE), and supporting tooling to issue those identities (SPIRE), that allows components in a distributed system to ensure the authenticity, integrity and confidentiality of messages passed between them. To accomplish it effectively, the supporting toolchain must be able to deliver identities to workloads across a range of hosting platforms (eg. private estate, public cloud) and runtime orchestration tools (eg. bare metal machines, virtual machines, containers).
+
```

Comment:
VMs or containers are not orchestration tools, they are computing environments for applications.

Reply:
The subject in the first clause is *runtime orchestration tools* not *orchestration tools* and the clause within parenthesis is meant to state the different runtimes that different orchestrators might deploy and lifecycle manage. I added additional words to make the meaning more explicit.